e2e/cypress/integration/multi_team_and_dm/gm_add_first_user_spec.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable jquery/no-text */
+/* eslint-disable jquery/no-find */
+
+describe('Multi-user group messages', () => {
+    const userPrefix1 = '0aadam';
+    const userPrefix2 = '0aabadam';
+    const userPrefix3 = 'beatrice';
+    const searchTerm = '0aa';
+    let testUser;
+    let testTeam;
+    before(() => {
+        // # Create a new team
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+
+            // # Add 3 users to the team that should be alphabetically sorted
+            createTestUser(userPrefix1, team);
+            createTestUser(userPrefix2, team);
+            createTestUser(userPrefix3, team);
+        });
+    });
+
+    it('MM-T463 Should not be able to create a group message with more than 8 users', () => {
+        // # Login as test user
+        cy.apiLogin(testUser);
+
+        // # Go to town-square channel
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the 'Direct messages' dialog
+        cy.get('#addDirectChannel').
+            click();
+
+        // # Start typing part of a username that matches previously created users
+        cy.get('#selectItems input[type="text"]').
+            type(searchTerm);
+
+        // * Expect user list to only contain usernames matchin the query term and to be sorted alphabetically
+        expectUserListSortedAlphabetically(searchTerm);
+
+        // # With the arrow and enter keys, select the first user that matches our search query
+        cy.get('body').
+            type('{downarrow}').
+            type('{enter}');
+
+        // * Expect to have the selected user's username to be displayed inside the search input
+        cy.get('.react-select__multi-value').
+            should('be.visible').
+            and('have.length', 1).
+            then(($selectedUser) => {
+                // # Get the selected user's username
+                const selectedUserName = $selectedUser.text();
+
+                // * Expect user list to not contain selected user, not be filtered, but still in alphabetical order
+                expectUserListSortedAlphabetically(selectedUserName, true);
+            });
+
+        // * Expect to have a remove button ('x') present next to the selected username (previous step)
+        cy.get('.react-select__multi-value__remove').
+            should('be.visible').
+            and('have.length', 1);
+
+        // * Expect search bar to have focus
+        cy.get('#selectItems input[type="text"]').should('have.focus');
```

Comment:
I was not able to match specific assertion`to the left of the cursor` in 
> Selected username appears in the search box to the left of the cursor 

as per https://automation-test-cases.vercel.app/test/MM-T459

There might be a way to get the cursor position with the `selectionStart` property of HTMLInputElement but this ReactComponent seems to do some tricks by hiding the input and displaying stuff in a `div` instead so my initial tests were inconclusive. I was not sure if it was worth pursuing so I decided to focus on the other assertions needed for this test instead. 

Should I try to make it work anyways ?

Reply:
Yes ! Awesome it works